### DISCUSSION TITLE
Quicker access to documents (for uploading)

### DISCUSSION THREAD

**Discussion by tpluscode** [inception-project/inception#5084] [Q&A]
The default left-hand menu contains links to Annotation, Curation, KB, Monitoring and Settings. However, the documents are only accessible via the admin panel

Is it possible to also show the documents on the dashboard?

**Comment by reckart** [comment]
The documents recently worked on should be listed on the right. 

Some projects contain thousands of documents. IMHO working with such a list in the open-documents dialog makes sense. Embedding the power of that dialog on the dashboard seems a bit of an overload to me.

Why would you like to have the ability to access documents directly from the dashboard?

**Reply by tpluscode** [reply]
I find it mildly annoying that when I upload documents I have go through the admin dashboard every time.

**Reply by reckart** [reply]
To reduce this annoyance, a button has been added to the right dashboard sidebar that provides direct access to the upload page if a project does not yet contain any documents. Did you notice that already?

**Reply by tpluscode** [reply]
Well yes, if a project does not contain any documents. To add more, I need to take the regular route

**Comment by tpluscode** [comment]
Speaking of the "Recent activity" panel, I was actually thinking if it would make sense to also add most recently uploaded documents there so that once uploaded, they can be immediately picked up for annotation.
